modules/fts/pages/fts-search-facets-term-facet.adoc

```diff
@@ -0,0 +1,56 @@
+[#term-facet]
+= Term Facet
+
+A term facet counts up how many of the matching documents have a particular term in a particular field.
+
+Most of the time, this only makes sense for relatively low cardinality fields, like a type or tags. It would not make sense to use it on a unique field like an ID.
+
+NOTE: When building a term facet, use the keyword analyzer. Otherwise, multi-term values get tokenized, and you cannot see the expected results.
```

Comment:
"...and you get unexpected results" sounds better than "and you cannot see the expected results"